docs/cadence/language/access-control.md

```diff
@@ -413,6 +413,79 @@ entitlement mapping M {
 
 attempting to map `(A | D)` through `M` will fail, since `A` should map to `(B, C)` and `D` should map to `E`, but these two outputs cannot be combined into a disjunctive set.
 
+### The `Identity` Mapping
+
+There is a `Identity` mapping built-in to Cadence that maps every input to itself as the output. 
+I.e., any entitlement set passed through the `Identity` map will come out unchanged in the output. 
+So, for example, for some resource defined like so:
+
+```cadence
+entitlement X
+
+resource SubResource {
+  // ...
+}
+
+resource OuterResource {
+    access(Identity) let childResource: @SubResource
+
+    access(Identity) getChildResource(): auth(Identity) &SubResource {
+      return &self.childResource
+    }
+
+    init(r: @SubResource) {
+        self.childResource = r
+    }
+}
+```
+
+Given some reference `ref` to `OuterResource` of type `auth(X) &OuterResource`, 
+accessing `ref.childResource` will yield a reference to `SubResource` of type `auth(X) &SubResource`. 
+
+One important point to note about the `Identity` mapping, however, is that because its full output range is unknown (and theoretically infinite),
+accessing an `Identity`-mapped field or function with an owned value will yield an empty output set. I.e. calling `getChildResource()` on an owned
+`OuterResource` value, for example, will produce an unauthorized `&SubResource` reference. 
+
+### Mapping Composition
+
+In the definition of an entitlement mapping, it is possible to `include` the definition of one or more other mappings, to "inherit" their mapping relations. 
```

Comment:
> "inherit" their mapping relations

Not sure if inheriting is the best term here, as it doesn't create an inheritance relationship between the two entitlements. It is more like a "copy over".